### PR TITLE
Hw 03 ivanovych

### PR DESCRIPTION
**Pull request by OI-22**
Hello!
Look and approve pull request, please.

### CODE REVIEW

File: HW_03_Ivanovych/task02.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Implement a calculator that translates the amount of information entered by the user in bytes into kilobytes and megabytes
+
+Size_in_bytes = int(input('Enter size (bytes): '))
```
Comment (katemalash) [inline HW_03_Ivanovych/task02.py:3]: Why is it from the capital letter?

File: HW_03_Ivanovych/task04.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#Calculate the area and perimeter of the circle over an entered radius
+import math
+
+
+radius = int(input('Enter radius of the circle'))
+
+area = math.pi * (radius ** 2)
+perimeter = (2 * (math.pi * radius))
```
Comment (katemalash) [inline HW_03_Ivanovych/task04.py:8]: Are the braces necessary in these two lines?

**Review by katemalash — APPROVED** [review]
Looks good. Please merge this pull request

File: HW_03_Ivanovych/task04.py (hunk 2 above)
Comment (OI-22) [reply HW_03_Ivanovych/task04.py]: taking away brackets

**Pull request merged**